ansible/library/port_alias.py

```diff
@@ -182,10 +183,15 @@ def main():
                 num_asic = multi_asic.get_num_asics()
             except Exception, e:
                 num_asic = 1
+        # if m_args['include_internal'] is not None:
+        #     include_internal = m_args['include_internal']
```

Comment:
These looks like debug code that should be removed now?

Reply:
Resolved